src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/AuditingTests.cs

```diff
@@ -228,6 +228,20 @@ public void TestBlobAuditingOnServer()
             RunPowerShellTest("Test-BlobAuditingOnServer");
         }
 
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.Sql)]
```

Comment:
@yoavrubin Why are these not attributed as check-in tests?  Please make the second category Category.CheckIn